### PR TITLE
Add onboarding-mizuki cluster argocd

### PR DESCRIPTION
**Pull request by eseiker**
follows #1249

### CODE REVIEW

File: 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    tag: 91
+
+  appProtocolVersion: "200131/eE394bb942fa7c2d807C170C7Db7F26cb3EA037F/MEQCIAiOFze6U9KZ12SLNKGyGGP.JMQMfoNbDppBq1kaByzUAiBv48M5eAKB695KpexNWuUmVfvWi3CWzVcEu2BSBNsMqA==/ZHU5OnRpbWVzdGFtcHUxMDoyMDI0LTAxLTA0ZQ=="
+  genesisBlockPath: "https://planets.nine-chronicles.com/planets/0x000000000001/genesis"
+  trustedAppProtocolVersionSigner: "031c5b9cb11b1cc07f8530599fa32338967e41cb364cca552a34ad2157ccb237bf"
+  
+  peerStrings:
+  - "03380b4ba8722057c9b4d8594f8de9481eb296aba4b3c168666f57b17596452ae7,heimdall-seed-1.nine-chronicles.com,31234"
```
Comment (area363) [inline 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml:19]:
```suggestion
  - "03380b4ba8722057c9b4d8594f8de9481eb296aba4b3c168666f57b17596452ae7,tcp-seed-1.9c-mizuki-heimdall.svc.cluster.local,31234"
```

File: 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+externalSecret:
+  enabled: true
+  prefix: 9c-onboarding-mizuki
+
+seed:
+  count: 1
+  useTurnServer: false
+
+  hosts:
+  - "heimdall-seed-1.nine-chronicles.com"
```
Comment (area363) [inline 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml:30]:
```suggestion
  - "tcp-seed-1.9c-mizuki-heimdall.svc.cluster.local"
```

File: 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  extraArgs:
+  - --tx-quota-per-signer=1
+
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: onboarding-mizuki-m7g_2xl_2c
+
+
+validator:
+  count: 4
```
[the remark this answers is not in the document]
Comment (eseiker) [reply 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml:60]: would current hostnames (validator-5~) be fine to be left as is?

[the remark this answers is not in the document]
Comment (area363) [reply 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml:60]: yes that's fine for now.

File: 9c-onboarding-mizuki/argocd/bootstrap.yaml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+          server:
+            nodeGroup: general-r6g_xl_2c
+            extraScrapeConfigs:
+              - job_name: scrape-headlesses
+                metrics_path: /metrics
+                scrape_interval: 8s
+                scrape_timeout: 6s
+                static_configs:
+                  - targets:
+                    - validator-5.9c-network.svc.cluster.local:80
```
Comment (sky1045) [inline 9c-onboarding-mizuki/argocd/bootstrap.yaml:45]: `validator-5.5.9c-mizuki-heimdall.svc.cluster.local:80` not critical

**Review by sky1045 — APPROVED** [review]

File: 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  - "validator-6.9c-mizuki-heimdall.svc.cluster.local"
+  - "validator-7.9c-mizuki-heimdall.svc.cluster.local"
+  - "validator-8.9c-mizuki-heimdall.svc.cluster.local"
+
+  extraArgs:
+  - --tx-quota-per-signer=1
+  - --consensus-target-block-interval=6500
+
+  nodeSelector:
+    eks.amazonaws.com/nodegroup: general-m7g_2xl_2c
```
Comment (area363) [inline 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml:80]:
```suggestion
    eks.amazonaws.com/nodegroup: onboarding-mizuki-m7g_2xl_2c
```

**Review by area363 — APPROVED** [review]

File: 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    repository: planetariumhq/ninechronicles-headless
+    tag: 91
+
+  appProtocolVersion: "200131/eE394bb942fa7c2d807C170C7Db7F26cb3EA037F/MEQCIAiOFze6U9KZ12SLNKGyGGP.JMQMfoNbDppBq1kaByzUAiBv48M5eAKB695KpexNWuUmVfvWi3CWzVcEu2BSBNsMqA==/ZHU5OnRpbWVzdGFtcHUxMDoyMDI0LTAxLTA0ZQ=="
+  genesisBlockPath: "https://planets.nine-chronicles.com/planets/0x000000000001/genesis"
+  trustedAppProtocolVersionSigner: "031c5b9cb11b1cc07f8530599fa32338967e41cb364cca552a34ad2157ccb237bf"
+  
+  peerStrings:
+  - "033369e95dbfd970dd9a7b4df31dcf5004d7cfd63289d26cc42bbdd01e25675b6f,tcp-seed-1,31234"
+
```
Comment (area363) [inline 9c-onboarding-mizuki/9c-mizuki-heimdall/values.yaml:20]:
```suggestion
snapshot:
  path: "main/heimdall/partition"

```

**Pull request merged**